source/_posts/2021-11-12-100k-analytics.markdown

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "100,000 installations in analytics!"
+description: "Our opt-in analytics have reached their first milestone: 100k users."
+date: 2021-11-12 00:00:00
+date_formatted: "November 12, 2021"
+author: Paulus Schoutsen
+author_twitter: balloob
+comments: true
+categories:
+- Announcement
+og_image: /images/blog/2021-11-analytics/social.png
+---
+
+<img src='/images/blog/2021-11-analytics/social.png' style='border: 0;box-shadow: none;'>
+
+Today we have reached a major milestone for [Home Assistant Analytics][analytics]: 100,000 users have opted in to be included!
+
+This is a big deal because it's not enabled by default, users have to opt-in, and we only launched Home Assistant Analytics last April. We promoted it in the release blog post and during live streams, but have never nagged existing users in the interface. New users did get asked to opt-in as part of onboarding.
+
+The 100,000 installations are not the total number of Home Assistant users. The truth here is that we don't know the total number of installations because Home Assistant is private by design. Our estimate is that there are 4-5x more installations than people that opt-in to analytics.
+
+_For the latest version of the graphs in this post, visit [Home Assistant Analytics][analytics]_
+
+![Line graph showing active installations over time hitting 100000 installations.](/images/blog/2021-11-analytics/active-installations.png)
+
+<!--more-->
+
+## Why we collect data
+
+It's good to start off with why we allow users to opt-in to share some data with us.
+
+Manufacturers of IoT products live in their own world in which we have little presence. We don't have stands at CES or other trade shows. Instead, manufacturers see we're open source and assume it's a small user base. They assume the ability to integrate with Home Assistant and other platforms is not profitable enough to allocate resources to add the necessary APIs to integrate.
+
+With the data collected by Home Assistant Analytics we can show that providing an API can expand their market around the globe.
+
+<p class='img'>
+<img src='/images/blog/2021-11-analytics/onboarding-analytics.png'>
+Screenshot of analytics during Home Assistant onboarding
+</p>
+
+## How people run Home Assistant
+
+The collected data is also used to give us, the Home Assistant developers, insight in how Home Assistant is installed and what we should improve. As you can see in our installation breakdown, the operating system installation type is by far the most popular. This makes sense because it is our recommended installation type and the easiest to use. It gives the full Home Assistant experience and can be fully managed via the user interface. No text editors, Linux experience or computer science degree necessary.
```

Comment:
```suggestion
The collected data is also used to give us, the Home Assistant developers, insight into how Home Assistant is installed and what we should improve. As you can see in our installation breakdown, the operating system installation type is by far the most popular. This makes sense because it is our recommended installation type and the easiest to use. It gives the full Home Assistant experience and can be fully managed via the user interface. No text editors, Linux experience, or computer science degree necessary.
```